src/test/regress/bin/psql-bt.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import subprocess, sys, os
+
+args = " ".join(map(lambda s: '"%s"' % s if ' ' in s else s, sys.argv[1:]))
+
+psql = subprocess.Popen([args + " 2>&1"], stdin=subprocess.PIPE, stdout=subprocess.PIPE, env=os.environ, shell=True)
```

Comment:
`2>&1` can instead be `stderr=subprocess.STDOUT` https://stackoverflow.com/questions/11495783/redirect-subprocess-stderr-to-stdout

Reply:
Yeah, the problem was that logs are output to stderr and everything else are output to stdout. In the final output of the psql process we need to output both, and getting ordering of two correct manually was more difficult than `2>&1`.

In the gdb process the ordering doesn't matter for us, so I did as you suggested.